nixos/modules/system/activation/activation-script.nix

```diff
@@ -181,7 +181,13 @@ in
           exit $_status
         '';
       };
+    };
 
+    system.showSystemDiffOnRebuild = mkOption {
+      type = lib.types.bool;
+      default = false;
+      example = true;
+      description = lib.mdDoc "Wether to diff the system against the currenly activated one on a NixOS system activation.";
```

Comment:
You can use `mkEnableOption (mdDoc "...")` here

Reply:
Then I would have ``Whether to enable`` as a prefix. I am fine with rewording this, just suggest something, but I didn't like anything that started with string.

Reply:
"Whether to enable diffing the system against the currently activated one on a NixOS system activation" should work fine